apps/wallet/src/ui/app/pages/site-connect/index.tsx

```diff
@@ -112,23 +116,23 @@ function SiteConnectPage() {
 					<UserApproveContainer
 						origin={permissionRequest.origin}
 						originFavIcon={permissionRequest.favIcon}
+						permissions={permissionRequest.permissions}
 						approveTitle="Connect"
 						rejectTitle="Reject"
 						onSubmit={handleOnSubmit}
 						approveDisabled={!accountsToConnect.length}
 						blended
 					>
 						<PageMainLayoutTitle title="Approve Connection" />
-						<SummaryCard
-							header="Permissions requested"
-							body={<DAppPermissionsList permissions={permissionRequest.permissions} />}
-							boxShadow
-						/>
-						<WalletListSelect
-							title="Connect Accounts"
-							values={accountsToConnect}
-							onChange={setAccountsToConnect}
-							boxShadow
+
+						<AccountMultiSelectWithControls
```

Comment:
Based on the design we need to split the accounts to unlocked and locked. Allow the user to connect only unlocked accounts and also have another section where user can unlock any locked accounts

Reply:
added this - will also update in follow-up PR for account locking / unlocking